composables/useUserAddresses.ts

```diff
@@ -5,8 +5,12 @@ import { useState, useNuxtApp } from "#app"
 export const useUserAddresses = () => {
   const nuxt = useNuxtApp()
   const fetcher = nuxt.nuxt2Context.$gqlFetch
-  const addresses = useState<Maybe<Array<CustomerContactCollection>>>(
-    `use-user-address-result`,
+  const userShippingAddresses = useState<Maybe<Array<CustomerContactCollection>>>(
```

Comment:
The types for both of the state references should be Maybe<Arrary<CustomerContact>>

Reply:
Resolved.